### PR TITLE
Fix regression on RedHat compilers

### PR DESCRIPTION
**Pull request by cottsay**
Regression caused by 26cdffa6e7cd64fc0dde9583e46341d1f1694d51

Example break: http://csc.mcs.sdsmt.edu/jenkins/job/ros-jade-euslisp_binaryrpm_22_i386/13/consoleFull

Thanks,

--scott

### CODE REVIEW

File: patches/CMakeLists.txt [hunk 1]
```diff
@@ -7,10 +7,10 @@ find_package(catkin)
 # check arch and os
 execute_process(COMMAND bash -c "gcc -dumpmachine" OUTPUT_VARIABLE gcc_dump_machine OUTPUT_STRIP_TRAILING_WHITESPACE)
 message("-- gcc dumpmachine returns ${gcc_dump_machine}")
-if(gcc_dump_machine MATCHES "x86_64-linux-.*")
+if(gcc_dump_machine MATCHES "x86_64-linux-.*" OR gcc_dump_machine MATCHES "x86_64-redhat-linux")
```
Comment (k-okada) [inline patches/CMakeLists.txt:10]: great, how about `gcc_dump_machine MATCHES "x86_64-.*linux-.*` ?
will this work for you?

Comment (cottsay) [inline patches/CMakeLists.txt:10]: It will not. The string is exactly `x86_64-redhat-linux`, no dash after `linux`.

**Comment by cottsay** [comment]
This is still broken. @k-okada, is this fix acceptable?

**Comment by k-okada** [comment]
Sorry, I just forgot about this. I will release new version within today

**Comment by k-okada** [comment]
OK, I have released new version 9.15.1

**Comment by cottsay** [comment]
Thanks!

**Pull request merged**